app/helpers/apo_helper.rb

```diff
@@ -1,14 +1,23 @@
 # frozen_string_literal: true
 
 module ApoHelper
-  # Retrieve a list of workflow templates from  the workflow service and return
-  # an array suitable for select_tag options
+  # @return [Array<Array<String, String>] array suitable for select_tag options
   def workflow_options
     Rails.cache.fetch 'workflow-templates-select-options' do
-      list = WorkflowClientFactory.build.workflow_templates
-      list.map do |name|
-        [name, name]
-      end
+      # per https://github.com/sul-dlss/argo/issues/3741, this should be hardcoded
+      [
+        # rubocop:disable Style/WordArray
+        ['accessionWF', 'accessionWF'],
+        ['gisAssemblyWF', 'gisAssemblyWF'],
+        ['gisDeliveryWF', 'gisDeliveryWF'],
+        ['gisDiscoveryWF', 'gisDiscoveryWF'],
+        ['goobiWF', 'goobiWF'],
+        ['registrationWF', 'registrationWF'],
+        ['wasCrawlDisseminationWF', 'wasCrawlDisseminationWF'],
+        ['wasCrawlPreassemblyWF', 'wasCrawlPreassemblyWF'],
+        ['wasSeedPreassemblyWF', 'wasSeedPreassemblyWF']
+        # rubocop:enable Style/WordArray
+      ]
     end
```

Comment:
small niggle, but to avoid having each name twice and risk having a typo, how about leave the existing `.map` block and just make `list = %w[accessionWF gisAssemblyWF etc etc]` in line 8?  

also, shouldn't need the Rails cache anymore since it's just a hardcoded array now

Reply:
good ideas;  didn't think of them.  will fix.